### DISCUSSION TITLE
Unstructured local libraries

### DISCUSSION THREAD

**Discussion by ItzCrazyKns** [superagent-ai/super-rag#82] [General]
Hi, In my implementation (https://github.com/ItzCrazyKns/Super-rag) I've added the support for unstructured libraries, currently it doesn't supporting anything like maximum number of characters as far as I can see in their repository (https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/partition/pdf.py).

**Comment by homanp** [comment]
@simjak do you know anything about this?

**Reply by simjak** [reply]
@ItzCrazyKns if you want to use Unstructured splitter with their OSS, you have to choose https://unstructured-io.github.io/unstructured/ingest/configs/chunking_config.html or https://github.com/Unstructured-IO/unstructured/blob/b6c1882cc3b813a547e65143515f831d1399a2ed/unstructured/chunking/title.py#L29

**Reply by ItzCrazyKns** [reply]
Thanks
